src/lib/server/paircmp.c

```diff
@@ -215,7 +215,7 @@ static int generic_cmp(UNUSED void *instance,
 
 		if (xlat_eval(value, sizeof(value), request, name, NULL, NULL) < 0) return 0;
 
-		MEM(vp = fr_pair_afrom_da(req, check->da));
+		MEM(vp = fr_pair_afrom_da(*request_list, check->da));
```

Comment:
this isn't right... not sure the original code was right either.  I think this should be `request->packet`

Reply:
I attempted to make the function parameters more consistent in the different comparison functions - so in this instance req was renamed to request_list.  So my change here retains existing functionality.